mdm/api/src/main/java/com/gentics/mesh/core/data/job/HibMailJob.java

```diff
@@ -0,0 +1,21 @@
+package com.gentics.mesh.core.data.job;
+
+/**
+ * Extension of {@link HibJob} for mail sending jobs.
+ */
+public interface HibMailJob extends HibJob {
+
+	/**
+	 * Get the mail messages.
+	 * @return
+	 */
+	String getMail();
```

Comment:
These method does not seem to be useful. Better to store the mail parts distinctly, as done in the sending context. This will allow the client to fetch/filter/reuse the jobs, if required.